docs/how-to/features/selective-forwarding.mdx

```diff
@@ -0,0 +1,337 @@
+
+```ts
+import { FishjamClient, type TrackId } from "@fishjam-cloud/js-server-sdk";
+const fishjamClient = new FishjamClient({
+  fishjamId: process.env.FISHJAM_ID!,
+  managementToken: process.env.MANAGEMENT_TOKEN!,
+});
+
+const room = await fishjamClient.createRoom({
+  roomType: "conference",
+});
+
+const { peer: publisher } = await fishjamClient.createPeer(room.id, {
+  subscribeMode: "auto",
+});
+
+const { peer: viewer } = await fishjamClient.createPeer(room.id, {
+  subscribeMode: "manual",
+});
+
+// ---cut---
+// Get track IDs from your room state or webhook events
+const audioTrackId = "track_audio_abc" as TrackId;
+const screenTrackId = "track_screen_xyz" as TrackId;
+
+// Subscribe to a single track
+await fishjamClient.subscribeTracks(room.id, viewer.id, [audioTrackId]);
+
+// Or multiple
+await fishjamClient.subscribeTracks(room.id, viewer.id, [
+  audioTrackId,
+  screenTrackId,
+]);
+```
+
+  </TabItem>
+
+  <TabItem value="python" label="Python">
+
+```python
+audio_track_id = "track_audio_abc"
+screen_track_id = "track_screen_xyz"
+
+# Subscribe to one track
+fishjam_client.subscribe_tracks(room.id, viewer_id, [audio_track_id])
+
+# Or multiple
+fishjam_client.subscribe_tracks(room.id, viewer_id, [audio_track_id, screen_track_id])
+```
+
+  </TabItem>
+</Tabs>
+
+> 📘 Track IDs are available from [Fishjam’s events in your backend](/how-to/backend/server-setup#listening-to-events).
+
+---
+
+## Step 4 — Combine multiple publishers
+
+A manual peer can subscribe to **several** publishers at once.  
+Each call to `subscribePeer` adds more publishers — it doesn’t overwrite the previous ones.
+
+<Tabs groupId="language">
+  <TabItem value="ts" label="Typescript">
+
+```ts
+import { FishjamClient, type PeerOptions } from "@fishjam-cloud/js-server-sdk";
+const fishjamClient = new FishjamClient({
+  fishjamId: process.env.FISHJAM_ID!,
+  managementToken: process.env.MANAGEMENT_TOKEN!,
+});
+
+const room = await fishjamClient.createRoom({
+  roomType: "conference",
+});
+
+// ---cut---
+const options = { subscribeMode: "manual" } satisfies PeerOptions;
+
+const { peer: viewer } = await fishjamClient.createPeer(room.id, options);
+const { peer: publisherA } = await fishjamClient.createPeer(room.id, options);
+const { peer: publisherB } = await fishjamClient.createPeer(room.id, options);
+
+// Subscribe viewer to two publishers
+await fishjamClient.subscribePeer(room.id, viewer.id, publisherA.id);
+await fishjamClient.subscribePeer(room.id, viewer.id, publisherB.id);
+```
+
+  </TabItem>
+
+  <TabItem value="python" label="Python">
+
+```python
+fishjam_client.subscribe_peer(room.id, viewer_id, publisher_a_id)
+fishjam_client.subscribe_peer(room.id, viewer_id, publisher_b_id)
+```
+
+  </TabItem>
+</Tabs>
+
+The viewer now receives tracks from **both** publishers simultaneously.
+
+---
+
+## Step 5 — Example: curated stage & audience
+
+You can implement a “stage” system where:
+
+- speakers are in auto mode (they see/hear each other),
```

Comment:
This means that speakers would receive all tracks from the audience, is this anticipated?